client/src/components/GraphViewerComponent/GraphViewerCytoscapeComponent/GraphViewerCytoscapeComponent.js

```diff
@@ -30,7 +30,9 @@ export class GraphViewerCytoscapeComponent extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      hideContextMenu: false
+      hideContextMenu: false,
+      initialX: 100,
+      initialY: 100
```

Comment:
What happens if there is something already at this point? Do we need to query the graph to see if there's a node already at this point?

Reply:
It cannot be anything at this point, the fix was required for new nodes without a preloaded layout.

Reply:
Will this logic run for graphs with existing nodes?

What if someone moves a node +50px and then creates another one?

Reply:
This one is already fixed on the last commit using the logic proposed from Jamie